web/web.go

```diff
@@ -111,6 +112,8 @@ func NewWebService(o *WebServiceOptions) *WebService {
 		router.Post("/-/quit", ws.quitHandler)
 	}
 
+	router.Get("/federate", instr("federate", o.Federation))
```

Comment:
I was wondering whether it should be part of `/api/v1`? But maybe there are good reasons against it? My thinking goes a bit like: if it's an endpoint that exposes machine-readable format, it should be part of the API. With the exception of `/metrics`, which is special.